### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by harsh0620**
- [1] I'm the package's author and/or maintainer.
- [1] I have have read [the docs][1].
- [1] My package repo has a description and a README describing what it's for and how to use it.
- [1] Any commands are available via the command palette.

My package is 'Snippet Cpp' which is the extension of snippets for sublime in c++ with the help of which one can do fast coding in Competitive Coding. I have also added all the Algorithms and Ways to tackle Data Structure like Tree, BST, and Graph so that while solving any problem one does not have to google the code if they know the concept. 

My package is similar to https://packagecontrol.io/packages/CPP%20Competitive%20Programming%20Snippets, However, it should still be added because I added more snippets that are necessary for the Competitive Programming as well as fast coding of Data Structure and Algorithm. On top of that, the quoted package has only a single snippet for any Data Structure but I added all the Algorithms and Ways to tackle Data Structure. You can check Graph, Tree, and BST sections.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
Please let us know why you're proposing this change.

**Comment by harsh0620** [comment]
I want to create a sublime package and I am following this documentation(https://packagecontrol.io/docs/submitting_a_package) for the same. While following the steps I have to get the pr request to merge, that's why I am proposing this change.

**Comment by braver** [comment]
Ah, that's not how you add a package. Add your entry to [repository/s.json](https://github.com/wbond/package_control_channel/blob/master/repository/s.json). And then come back to [complete the checklist](https://github.com/wbond/package_control_channel/blob/master/.github/PULL_REQUEST_TEMPLATE.md) and explain why your package should be added.

**Comment by harsh0620** [comment]
I added the file as per your reply and also updated the checklist. Please have a look and suggest to me any changes that I have to do add this package.

**Comment by braver** [comment]
Please don't format the whole file, and please remove your entry from channel.json.

**Comment by harsh0620** [comment]
Removed my entry from the channel.json

**Comment by braver** [comment]
> Please don't format the whole file

Your changes cover the whole file. Please just add your package without also formatting everything else.

**Comment by harsh0620** [comment]
Please look if the change is now ok. It is the same as in the main s.json file of this repo but with the addition of my package.

**Comment by braver** [comment]
Alright, that's better. Still though, why can't your changes be added to the existing package? That would benefit all current and potential future users of that package.

**Comment by harsh0620** [comment]
So it's the end. I will not be able to add my package on sublime either that package is not yet updated after its publishing in 2017.

**Comment by braver** [comment]
Have you tried opening a PR on that existing package? If that doesn't work out we can consider replacing it with yours.

**Comment by harsh0620** [comment]
I tried but there is no revert back from the owner.

**Comment by harsh0620** [comment]
Can you please now guide me to the next steps?

**Comment by braver** [comment]
You can try pinging them here in this PR, and then we’ll give them 2 weeks to respond. After that we can go ahead and replace the existing package and bring the improvements to all existing users.

**Comment by harsh0620** [comment]
I tried pinging several times for two weeks but there is no reply so now we can go-ahead?

**Comment by braver** [comment]
Indeed, the package from @MeghaSharma21 seems well and truly dead. The best way to go now would be to replace the existing package with yours. You can change the repo url in the existing entry (and make any other changes you’d like) instead of adding a new entry.

File: repository/s.json [hunk 1]
```diff
@@ -5906,4 +5917,4 @@
 			]
 		}
 	]
-}
+}
```
[the remark this answers is not in the document]
Comment (harsh0620) [reply repository/s.json]: Please have a look is the changes enough and at right place.

**Comment by harsh0620** [comment]
Please review the changes

**Comment by braver** [comment]
Again, please don’t format the entire file and review your proposed changes closely.

**Pull request closed without merging**